e2e-tests/constants/website-address-examples.js

```diff
@@ -0,0 +1 @@
+export const VALID_WEBSITE = 'www.google.com';
```

Comment:
Do you think we should have this structure?
```js
constants/examples/phone-number
constants/examples/website-address
```

Then could export as `LONG`, `INTERNATIONAL`, `VALID`?

Reply:
done